README.md

```diff
@@ -15,7 +15,9 @@ Required nodejs v0.10.29+ and npm v2.1.0+.
 Configuration
 -------------
 
-TODO
+You can configure the client using the following environment variables:
+
+- `READINGLIST_SERVER_BASEURL`: the Readinglist server base URL; default: `http://0.0.0.0:8000/v0`
```

Comment:
The deployment process will be:
- fetch the sources from the repo
- define config env var, eg. serveur base url as documented here
- build the dist files for the client app
- that build will reflect the configured base url using the env var

That simple :)